lib/active_admin/formatter/base.rb

```diff
@@ -0,0 +1,19 @@
+module ActiveAdmin
+  module Formatter
+    class Base
+      extend ActiveAdmin::Formatter::DSL
+
+      def initialize(object, context)
+        @object, @context = object, context
+      end
+
+      detection do |object, context|
```

Comment:
Unused block argument - `object`. You can omit all the arguments if you don't care about them.<br>Unused block argument - `context`. You can omit all the arguments if you don't care about them.